### DISCUSSION TITLE
[feature request] Independent Solid Infill/Sparse Infill Angle

### DISCUSSION THREAD

**Discussion by DaubsDesigns** [OrcaSlicer/OrcaSlicer#1123] [Ideas]
I love Orca Slicer! First time I used it, I never went back to SuperSlicer.. however.. 

I think it would be nice to be able to change the solid infill angle independently of the sparse infill angle.. 45deg looks a lot better on the top surfaces.. but with the infill pattern I'm using, a 0 deg sparse infill pattern would be significantly stronger. I've added pictures to describe what I mean. Not sure if others would find this feature useful.. but as someone that prints prototype parts for functional ideas.. I would definitely use this!

![image](https://github.com/SoftFever/OrcaSlicer/assets/47301648/a8a27a54-4091-4f2d-a8aa-8d4949cbe67d)
![image](https://github.com/SoftFever/OrcaSlicer/assets/47301648/dc69131c-70bb-46f3-94fc-dbe7d2082dbc)

**Comment by Roycinger** [comment]
+1 this.
I'm missing the option to rotate the top infill in general (or I'm unable to find the correct setting)
![image](https://github.com/SoftFever/OrcaSlicer/assets/7791335/341ee331-7d03-4d03-944a-f08e6e499592)
The top infill can't be rotated 90° here
